library/src/main/kotlin/me/thanel/swipeactionview/SwipeActionView.kt

```diff
@@ -810,6 +878,10 @@ class SwipeActionView : FrameLayout {
             return
         }
 
+        if(hasSwipedFarEnoughMultiContainerFirstElement(container.translationX)){
+            activate(container.translationX > 0, true)
+        }
+
         if (hasSwipedFarEnough(container.translationX) || swipedFastEnough) {
             activate(container.translationX > 0)
         } else {
```

Comment:
These checks could be combined to only trigger halfway swipe if we are sure that regular swipe didn't activate.

```suggestion
        if (hasSwipedFarEnough(container.translationX) || swipedFastEnough) {
            activate(container.translationX > 0)
        } else if (hasSwipedFarEnoughMultiContainerFirstElement(container.translationX)) {
            activate(container.translationX > 0, true)
        } else {
```